src/cli/javascript/mod.rs

```diff
@@ -1,15 +1,35 @@
+use std::collections::BTreeMap;
+pub use std::sync::Arc;
+
+use hyper::header::{HeaderName, HeaderValue};
+
+mod channel;
 mod http_filter;
+mod js_request;
+mod js_response;
 mod runtime;
-mod serde_v8;
-mod shim;
-pub use std::sync::Arc;
 
 pub use http_filter::HttpFilter;
+pub use js_request::JsRequest;
+pub use js_response::JsResponse;
 pub use runtime::Runtime;
 
 use crate::{blueprint, HttpIO};
 
 pub fn init_http(http: impl HttpIO, script: blueprint::Script) -> Arc<dyn HttpIO> {
+    log::debug!("Initializing JavaScript HTTP filter: {}", script.source);
     let script_io = Runtime::new(script);
     Arc::new(HttpFilter::new(http, script_io))
 }
+
+pub(crate) fn create_header_map(
```

Comment:
```suggestion
fn create_header_map(
```

Needs to be used only within the mod